File: src/main/kotlin/day00/solution.kt
package day00

import java.io.File

fun first(inp: String): Int {
    throw NotImplementedError()
}

fun second(inp: String): Int {
    throw NotImplementedError()
}

fun main() {
    val testInput = File("src/main/kotlin/day00/test-input.txt").readText()
    val input = File("src/main/kotlin/day00/input.txt").readText()
    println(first(testInput))
    println(second(testInput))
}

File: haskell/day00/solution.hs
first :: String -> Int
first inp = error "not implemented yet"

second :: String -> Int
second inp = error "not implemented yet"

main :: IO ()
main = do
  testInput <- readFile "../../src/main/resources/day00/test-input.txt"
  input <- readFile "../../src/main/resources/day00/input.txt"
  print $ first testInput
  print $ second testInput
